setup.py

```diff
@@ -36,7 +36,7 @@
     long_description=README,
     platforms="Posix; MacOS X",
     include_package_data=True,
-    install_requires=("protobuf >= 3.12.0",),
+    install_requires=("protobuf >= 3.19.0",),
```

Comment:
This should not have been included in a patch release. It's causing problems over in the conda ecosystem, where the version of `libprotobuf` is a bit slower to update as it require recompilation of many many packages. Currently the latest grcp-cpp package is compiled with `libprotobuf==3.18.1`.

Reply:
Update: literally today is when they are rolling out changes to the build environment to support protobuf 3.19 https://github.com/conda-forge/grpc-cpp-feedstock/pull/124

Reply:
For future reference, this is the where pin for conda was updated: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2049

I recommend we don't update minimum protobuf beyond this in future